### DISCUSSION TITLE
Welcome to Aporta Discussions!

### DISCUSSION THREAD

**Discussion by bytedreamer** [bytedreamer/Aporta#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeuxtest** [comment]
hello, 
I want to use you softwere. I have load it but can't to intall it. It dispay this message following (see document attached). Please can you help me. thank you. . 

![Error message](https://github.com/bytedreamer/Aporta/assets/133159545/9aa3da53-0815-4cb0-bd9c-361111008f88)

**Reply by bytedreamer** [reply]
A new version of the Aporta installer has been uploaded to the server that should resolve your issue

**Reply by jeuxtest** [reply]
Hello Bytedreamer,
Thank you for return. 
I installed the app successfully.
when i connect with localhost it works but when i want to connect with ip address i can't.
I want to connect the application to a device that I have recovered, the communication is done by TCP; Can you help me to solve the pb.
THANKS

**Reply by bytedreamer** [reply]
I had the same issue on my test system. The default Windows firewall is blocking the connection. If you disable the firewall it should connect. I will be adding an exception to the firewall in the next build. Thanks for the feedback.

**Reply by jeuxtest** [reply]
the connection problem is related to HTTPS. it is resolved.
However, my control box cannot connect to Aporta. see the screenshot of my last message.
my system is from Horoquartz.

**Reply by jeuxtest** [reply]
![image](https://github.com/bytedreamer/Aporta/assets/133159545/aab3f2af-985a-4a10-bdcf-613ae053fb07)

**Reply by jeuxtest** [reply]
your application display this message :
![image](https://github.com/bytedreamer/Aporta/assets/133159545/f66efc58-7b9b-4d9f-aabc-7bc2d3509afe)

**Reply by jeuxtest** [reply]
your application desplay me this message 

![image](https://github.com/bytedreamer/Aporta/assets/133159545/db501d68-c685-440d-a9ef-2d7635440f39)

**Reply by bytedreamer** [reply]
I will update this message. It should indicate the following

**Add Door** No access readers or inputs are configured

**Reply by bytedreamer** [reply]
A reader needs to be configured in OSDP driver

**Reply by jeuxtest** [reply]
It offers me to configure the RS-485 port (COM3 port). can't we use the RJ45 port and communicate in TCP?

**Reply by bytedreamer** [reply]
That option will be in a future version. Working to get the basic access control functionality completed first.

**Reply by jeuxtest** [reply]
hello, 
can you tell me what information should I put in the address box? 
![image](https://github.com/bytedreamer/Aporta/assets/133159545/d6d067b4-b772-4d99-8c1d-e90eb89c5115)

**Reply by bytedreamer** [reply]
It is the address of the OSDP device. You can use OSDP Bench to discover the address of the device. https://osdpworld.com/osdp-bench/

**Reply by jeuxtest** [reply]
Hello,
how are you. 
I still can't get your app to talk to my system. you will find in PJ the diagram of my system as well as its control panel. I connect aport on the COM3 of my system. but nothing happens.

![image](https://github.com/bytedreamer/Aporta/assets/133159545/8784b5a0-3d27-49d0-9618-975aec3aeb38)

**Reply by jeuxtest** [reply]
[description_badgeuse_2.pdf](https://github.com/bytedreamer/Aporta/files/11990541/description_badgeuse_2.pdf)

**Reply by bytedreamer** [reply]
I notice that the board has 4-wire RS-485. Make sure that it is connected properly to the reader's 2-wire connections.

https://support.tibbo.com/hc/en-us/articles/115009220808-How-to-connect-2-wire-RS485-to-4-wire-RS485

**Reply by mistial-dev** [reply]
> It is the address of the OSDP device. You can use OSDP Bench to discover the address of the device. https://osdpworld.com/osdp-bench/

Would it make sense to allow broadcast address for situations where there's only one device on the bus?

**Reply by bytedreamer** [reply]
The OSDP technical working group is working on adding discovery to the OSDP specification. I plan on adding it the Aporta.

**Comment by Oppfields** [comment]
It would be useful to list additional OSDP boards as the Cypress and Azure items are hard to locate in the UK. Also the Quick Start Guide assumes a basic knowledge of the hardware set up. I suspect there will be many users who are new to access control (myself included) and a basic hardware list and hook up diagram would be really beneficial for uptake and problem solving.

**Reply by bytedreamer** [reply]
I will work to add hardware wiring diagrams before there is an official v1.0 release. Thanks for the feedback.
